cfy_cluster_setup/logger.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import logging
+
+
+def setup_console_logger(verbose):
+    logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)
+
+    # paramiko.transport, invoke, and fabric are very verbose
+    if verbose < 2:
```

Comment:
Can't verbose just be a boolean?

Reply:
Oh, no, I see why you're doing that. However, shouldn't this check then be if verbose >= 2, since presumably that's the very verbose mode while < 2 means somewhat verbose but not very?

Reply:
In this case, we "hide" these logs since they are very verbose. So in case the verbose level is >=2 (very verbose), we will show them.